components/base/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react"
+import Paper from "@mui/material/Paper"
+import InputBase from "@mui/material/InputBase"
+import IconButton from "@mui/material/IconButton"
+import SearchIcon from "@mui/icons-material/Search"
+import "./SearchField.scss"
+
+type SearchFieldtype = {
+  handleChangeFilmTitle: (filmTitle: string | undefined) => void
+  filmName: string | undefined
+}
```

Comment:
```suggestion
type SearchFieldProps = {
  handleChangeSearch: (value: string | undefined) => void
  searchValue: string | undefined
}
```
поиск также может быть не привязан к фильмам, мы его будем юзать и в других каталогах